### DISCUSSION TITLE
不知道大家有没有在玩 Minecraft 1.5.2 及以下版本呢？

### DISCUSSION THREAD

**Discussion by zkitefly** [Meloong-Git/PCL#2181] [讨论]
有一个 Bug ，有惊喜qwq

**Comment by zkitefly** [comment]
https://github.com/Hex-Dragon/PCL2/issues/2182

**Comment by iPodtouch6557** [comment]
我一年前尝试开了infdev和alpha1.2.3_04的生存坑，早就弃了（）
我还体验过一些基于老版本开发的恐怖版, 在B站@普通の史蒂夫那里下载了好几个G的恐怖版本合集
